roles/redfish_monitor/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Pull Docker image.
+  docker_image:
+    source: pull
+    name: "{{ redfish_monitor_docker_image }}"
+    force_source: yes
+
+- name: Create config directory.
+  file:
+    path: "{{ redfish_monitor_config_path }}"
+    state: directory
+
+- name: Create log directory.
+  file:
+    path: "{{ redfish_monitor_log_folder }}"
+    state: directory
+
+- name: Configure redfish-monitor env file
+  template:
+    src: redfish-monitor.env.j2
+    mode: 0600
+    dest: "{{ redfish_monitor_env_file }}"
+
+- name: Start redfish-monitor docker container
```

Comment:
We normally provision these things as systemd services - there are pros and cons. What were your reasons for using the `docker_container` mechanism here?

Reply:
The asynchronous code requires python3 which we don't have configured on admin01.

Reply:
In the prometheus role we set up systemd services that call Docker: https://github.com/ChameleonCloud/chi-in-a-box/blob/master/roles/prometheus/templates/systemd.service.j2